typescript/infra/hardhat.config.ts

```diff
@@ -59,6 +61,10 @@ task('kathy', 'Dispatches random hyperlane messages')
       const recipient = await recipientF.deploy();
       await recipient.deployTransaction.wait();
 
+      const isAutomine: boolean = await hre.network.provider.send(
```

Comment:
What's going on here? Testing if the provider supports this?

Reply:
makes it agnostic to the node config